chp00_introduction/NOC_I_01_RandomWalkTraditional/sketch.js

```diff
@@ -5,9 +5,9 @@
 var walker;
 
 function setup() {
-  createCanvas(640,360);
-  walker = new Walker();
-  background(127);
+  createCanvas(640,360); //creating canvas of size 640 x 360
```

Comment:
I really appreciate these comments, amazing! I think I prefer the style where the comments are above the line of code, a space between the slashes and first character, and the first character capitalized, i.e.:

```javascript
// Creating canvas of size 640 x 360
createCanvas(640, 360); 
```

What do you think? Also, I believe there are comments from the book text that can be pulled into here. Ideally I'd like them to match.

Reply:
Ok will check them and update the pull request.

Reply:
Also without http or https. Simply loading html doesn't work. The user must consciously change the url in index.html file to either http or https and for someone who just want to see the code in action, this seems to be a hassle. I personally check official code only when my code is not working and I can't debug it. To debug the official code as well and correct it seems a bit tough on the person who is looking to find quick solution. In the end it's your call. I personally prefer either http / https in the link. That way even if I come 1 year later to see how code looks I can instantly see result and dive into the code of the one I am interested in. :smiley:

Reply:
Yes, but it does work if you run a local server which is a good habit to get into. But maybe this is too much of a pain for people?

Reply:
Oh my god. I never realized that. Probably gonna add a comment to read me in installation instructions. This is wonderful if you run a local server from the source folder. It looks just like a book. Maybe this is the reason why there are so many issues/PR for changing the links.People don't know about it. I could add instructions for running section and it may help you save some time and probably a contribution guideline as well to filter unnecessary PR. What do you say? @shiffman

Reply:
Yes, that would be great! Here are more resources:

https://github.com/processing/p5.js/wiki/Local-server
https://www.youtube.com/watch?v=HZ4D3wDRaec
https://www.youtube.com/watch?v=UCHzlUiDD10